lib/python/treadmill_aws/hostmanager.py

```diff
@@ -3,6 +3,7 @@
 import time
 import yaml
 
+from botocore.exceptions import ClientError
```

Comment:
do not import classes, we import modules and use qualified names for classes later on.

from botocore import exceptions as boto_exceptions

...

except botoexceptions.ClientError as err:
...